lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -125,35 +152,166 @@ withNetworkLayer tr project k = k NetworkLayer
+                fromIntegral _protocolParamsCoinsPerUtxoWord
+            stakeKeyDeposit = Coin $ fromIntegral _protocolParamsKeyDeposit
+            eras = emptyEraInfo
+            maximumCollateralInputCount =
+                fromIntegral _protocolParamsMaxCollateralInputs
+            minimumCollateralPercentage =
+                fromIntegral _protocolParamsCollateralPercent
+            executionUnitPrices = Just $ ExecutionUnitPrices
+                { pricePerStep = toRational _protocolParamsPriceStep
+                , pricePerMemoryUnit = toRational _protocolParamsPriceMem
+                }
+            currentNodeProtocolParameters = Just Node.ProtocolParameters
+                { protocolParamProtocolVersion =
+                    ( fromIntegral _protocolParamsProtocolMajorVer
+                    , fromIntegral _protocolParamsProtocolMinorVer
+                    )
+                , protocolParamDecentralization =
+                    toRational _protocolParamsDecentralisationParam
+                , protocolParamExtraPraosEntropy =
+                    Nothing
+                , protocolParamMaxBlockHeaderSize =
+                    fromIntegral _protocolParamsMaxBlockHeaderSize
+                , protocolParamMaxBlockBodySize =
+                    fromIntegral _protocolParamsMaxBlockSize
+                , protocolParamMaxTxSize =
+                    fromIntegral _protocolParamsMaxTxSize
+                , protocolParamTxFeeFixed =
+                    fromIntegral _protocolParamsMinFeeB
+                , protocolParamTxFeePerByte =
+                    fromIntegral _protocolParamsMinFeeA
+                , protocolParamMinUTxOValue =
+                    Just $ Node.Lovelace $ fromIntegral _protocolParamsMinUtxo
+                , protocolParamStakeAddressDeposit =
+                    fromIntegral _protocolParamsKeyDeposit
+                , protocolParamStakePoolDeposit =
+                    fromIntegral _protocolParamsPoolDeposit
+                , protocolParamMinPoolCost =
+                    fromIntegral _protocolParamsMinPoolCost
+                , protocolParamPoolRetireMaxEpoch =
+                    fromIntegral _protocolParamsEMax
+                , protocolParamStakePoolTargetNum =
+                    fromIntegral _protocolParamsNOpt
+                , protocolParamPoolPledgeInfluence =
+                    toRational _protocolParamsA0
+                , protocolParamMonetaryExpansion =
+                    toRational _protocolParamsRho
+                , protocolParamTreasuryCut =
+                    toRational _protocolParamsTau
+                , protocolParamUTxOCostPerWord =
+                    Just $ Node.Lovelace $
+                    fromIntegral _protocolParamsCoinsPerUtxoWord
+                , protocolParamCostModels =
+                    mempty
+                    -- Cost models aren't available via BF
+                    -- TODO: Hardcode or retrieve from elswhere.
```

Comment:
Yeah. 😕 Could you add this to a backlog, either your personal backlog or our backlog in JIRA, so that we don't forget about this?

Reply:
https://input-output.atlassian.net/browse/ADP-1572